src/hidapi/libusb/hid.c

```diff
@@ -716,7 +716,7 @@ static void invasive_fill_device_info_usage(struct hid_device_info *cur_dev, lib
 static struct hid_device_info * create_device_info_for_device(libusb_device *device, libusb_device_handle *handle, struct libusb_device_descriptor *desc, int config_number, int interface_num, int interface_class, int interface_subclass, int interface_protocol)
 {
 	struct hid_device_info *cur_dev = calloc(1, sizeof(struct hid_device_info));
-	if (cur_dev == NULL) {
+	if (!cur_dev) {
```

Comment:
We shouldn't touch the third-party code. 
We can probably reuse the exclude list from check_stdlib_usage.py:
```
    excluded_paths = [
        "src/stdlib",
        "src/libm",
        "src/hidapi",
        "src/video/khronos",
        "include/SDL3",
        "build-scripts/gen_audio_resampler_filter.c",
        "build-scripts/gen_audio_channel_conversion.c" ]
```

Reply:
Actually, we should include src/stdlib, but the others can be excluded.